frontend/src/components/MeetingRoom/ToolbarOverflowButton/ToolbarOverflowButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { ReactElement, useCallback, useRef, useState } from 'react';
+import { Tooltip } from '@mui/material';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+import ToolbarButton from '../ToolbarButton';
+import ToolbarOverflowMenu from '../ToolbarOverflowMenu';
+
+/**
+ * ToolbarOverflowButton Component
+ *
+ * Displays a clickable button that opens a grid of hidden toolbar buttons for smaller viewport devices.
+ * @returns {ReactElement} - The ToolbarOverflowButton Component.
+ */
+const ToolbarOverflowButton = (): ReactElement => {
+  const anchorRef = useRef<HTMLButtonElement>(null);
+  const [open, setOpen] = useState<boolean>(false);
+  const [openEmojiGridMobile, setOpenEmojiGridMobile] = useState<boolean>(true);
```

Comment:
We need to save the state outside of the emoji-grid components otherwise the preference on mobile of whether the grid is open or closed isn't saved.